backend/src/main/java/com/hyooun/wilson/review/domain/ReviewResult.java

```diff
@@ -0,0 +1,14 @@
+package com.hyooun.wilson.review.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.List;
+
+@Getter
+@AllArgsConstructor
+public class ReviewResult {
+
+	private final String summary;
```

Comment:
ReviewResult 클래스에 suggestions 필드 추가 필요.

```suggestion
private final List<String> suggestions;
```